docs/standard/using-linq.md

```diff
@@ -69,7 +69,7 @@ For a significant chunk of software out in the wild, everything revolves around
 Consider the following: finding all XML elements with a specific attribute value.
 
 ```cs
-public static FindAllElementsWithAttribute(XElement documentRoot, string elementName,
+public static IEnumerable<XElement> FindAllElementsWithAttribute(XElement documentRoot, string elementName,
                                            string attributeName, string value)
```

Comment:
What about:

```cs
public static IEnumerable<XElement> FindAllElementsWithAttribute(XElement documentRoot,
                                                                 string elementName,
                                                                 string attributeName,
                                                                 string value)
```

Also I don't think there's a problem with:

```cs
public static IEnumerable<XElement> FindAllElementsWithAttribute(XElement documentRoot, string elementName,
                                                                 string attributeName, string value)
```

The more advanced sample further down the page in the section **Essential Samples** also contains a scroll bar